tests/test_xsd.py

```diff
@@ -92,46 +108,58 @@ def test_xsd():
     """)
 
     xml_doc = run_and_parse(testdir).xml_doc
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd('mk8s')))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd('mk8s')))
```

Comment:
Since an external program is accessing this method it should be marked as public.